build.gradle

```
@@ -17,13 +17,19 @@ loom {
 }
 
 repositories {
-	// So far none...
+		maven { url "https://maven.shedaniel.me/" }
+		maven { url "https://maven.terraformersmc.com/releases/" }
 }
 
 dependencies {
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
 	mappings loom.officialMojangMappings()
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
+	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
+	modImplementation "com.terraformersmc:modmenu:4.0.6"
+	modApi("me.shedaniel.cloth:cloth-config-fabric:8.0.75") {
```

You could actually add Mod Menu and Cloth Config versions as variables like this:
```suggestion
	modApi("me.shedaniel.cloth:cloth-config-fabric:${project.cloth_config_version}") {
```
Don't forget to add those variables with values to gradle.properties as well